### PR TITLE
Add macOS ARM

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #4

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
So, Arm works but no Linux fails and I have no idea why.

**Comment by BastianZim** [comment]
@sstadick Have you seen something like this? Otherwise I'll ask in Gitter for help.

**Comment by sstadick** [comment]
@BastianZim No, I haven't seen that specific error before and it doesn't ring any bells. Since it's zlib it seems like this would be an issue many others have also hit though?

**Comment by BastianZim** [comment]
Hopefully – let's see.

https://gitter.im/conda-forge/conda-forge.github.io?at=620a7ddfb8b0824482341119

**Comment by xhochy** [comment]
You need `zlib` in `host`. It probably picks it up from `build` otherwise.

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
Thanks @xhochy

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,6 +20,9 @@ requirements:
     - pkg-config
     - make
     - cmake
+    - libgit2
```
Comment (xhochy) [inline recipe/meta.yaml:23]: This needs to be in host

Comment (BastianZim) [reply recipe/meta.yaml:23]: I had tried that but got a different error. Let me update it again because somehow the logs were deleted.

**Comment by BastianZim** [comment]
@isuruf Thanks for the fix but the error still occurs.

**Comment by BastianZim** [comment]
@isuruf It's still failing on Mac.

The image not found was reported a couple of times before but apparently just went away again at some point: https://github.com/conda-forge/conda-forge.github.io/issues/1574 No idea what the correct fix here is.

The ARM fault is the same as the initial error and I fixed it by including the if statement that you reversed in https://github.com/conda-forge/cargo-bundle-licenses-feedstock/pull/6/commits/78c3d392aa3c836a7baefba331346e6cf00ca675 if I understand it correctly. Do you know of a better fix, otherwise I'd just add that again.

**Comment by xhochy** [comment]
This is an issue with the `built` crate. The following  patch makes the build pass (disclaimer: I have no Rust knowledge and the code only compiles by trail and error).

```diff
diff --git a/Cargo.toml b/Cargo.toml
index df74e40..80e53e1 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,5 +37,5 @@ toml = "0.5.8"


 [build-dependencies]
-built = {version = "0.5.1", features = ["git2"]}
+built = {version = "0.5.1"}
 strum = {version = "0.21.0", features = ["derive"]}
diff --git a/src/main.rs b/src/main.rs
index db5e38e..1867943 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -22,13 +22,13 @@ pub mod built_info {
     ///   - Git dirty info (whether the repo had uncommitted changes)
     ///   - Cargo package version if no git info found
     fn get_software_version() -> String {
-        let prefix = if let Some(s) = GIT_COMMIT_HASH {
+        let prefix = if let Some(s) = Some("PKG_VERSION") {
             format!("{}-{}", PKG_VERSION, s[0..8].to_owned())
         } else {
             // This shouldn't happen
             PKG_VERSION.to_string()
         };
-        let suffix = match GIT_DIRTY {
+        let suffix = match Some(false) {
             Some(true) => "-dirty",
             _ => "",
         };
```

**Comment by BastianZim** [comment]
@isuruf Just to quickly check, would you like to test more approaches here or are you happy with me applying @xhochy's fix here? (Thanks for finding that by the way!)

**Comment by BastianZim** [comment]
@isuruf Gentle ping

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
Ping again @isuruf 

(Will have to create a new PR because of the merge conflicts though)

**Comment by isuruf** [comment]
I can have a look to fix this if you can merge the conflicts

**Comment by BastianZim** [comment]
Thank you!

Somehow merging fails – I'll open a new PR.

**Comment by BastianZim** [comment]
New PR is in #8

**Pull request closed without merging**